website/docs/cloud-docs/agents/request-forwarding.mdx

```diff
@@ -50,6 +51,13 @@ the agent starts, you will see the following log messages:
 These log messages indicate that the agent has successfully connected to HCP Terraform
 and is ready to start forwarding requests.
 
+### (Optional) Accept ingress jobs needed for Private VCS access
+By default, HCP Terraform will try to clone a repository using its own infrastructure.
```

Comment:
Rewriting this in a single comment to avoid splitting it up over lines. I attempted to shorten for brevity and make suggestions for active voice and present tense:
```suggestion
If you want to use a [private VCS provider](/terraform/cloud-docs/vcs/private), you must use the HCP Terraform agent's configuration version ingress to access those private repositories. For agents to accept the ingress job type, you must start your HCP Terraform agent with the `-accept` flag and the `ingress` value. Otherwise, your agent throws errors if you use a private VCS provider.
```

Reply:
This may not technically be correct though — tagging in @BrianMMcClain to review this too to ensure I got everything right!

Reply:
Yes, the reason I open this PR is because no error on agent side, refusal of ingress or any indication of misconfiguration is visible. Fetching configuration Version simply times out on the workspace side.

Reply:
According to our [corresponding docs for HCP Terraform](https://developer.hashicorp.com/terraform/cloud-docs/vcs/private#configure-the-agent-pool-and-agent) and some testing, there's two flags that need to be set for an agent to handle private VCS connections:

- `-accept` accepts a comma-separated list of values, one of which must be the value `ingress`
- `-request-forwarding` - Just including this flag is enough, but also setting it to `-request-forwarding=true` seems to work too

If you're running the `tfc-agent` binary directly, an example of this command would look like:

```
tfc-agent \
-name private-vcs-test-01 \
-request-forwarding=true \
-accept "ingress" 
```

In Docker, we set these values using the corresponding environment variables instead, so that command would look like this:

```
docker run -e TFC_AGENT_TOKEN -e TFC_AGENT_NAME -e TFC_AGENT_ACCEPT=ingress -e TFC_AGENT_REQUEST_FORWARDING=true hashicorp/tfc-agent:1.17.0
```

Reply:
Oh @henryk-rychlik can we just link to our docs on this instead? Under this header just having the following and not 
```suggestion
If you want to use a private VCS provider, you must configure your HCP Terraform agent to support private VCS by allowing access to those private repositories. To learn more, refer to [Connect to Private VCS Providers](/terraform/cloud-docs/vcs/private#configure-the-agent-pool-and-agent). 
```